.gitignore

```diff
@@ -3,6 +3,8 @@
 
 # Winflexbison specifics
 CMakeBuild*/
+#vscode
+.vscode
```

Comment:
This is now already added

Reply:
Those are actually project-specific settings that users can override. They only apply for the project, not outside of it.

> and one could easily add a malicious extension to it.

A config that adds a malicious extension would first need to get merged by the project, and the malicious extenion would first need to get published in the extension store. Just allowing those files in principle via the .gitignore without having those files does nothing so far. I think this should be discused further in #92 or somewhere else, as it is off-topic for this PR.